bin/run-jetpack-command.sh

```diff
@@ -25,9 +25,10 @@ nvm install
 listed_pnpm_version=$(npx -c 'echo $npm_package_engines_pnpm')
 pnpm_version=$(npx semver -c "$listed_pnpm_version")
 
-cd projects/plugins/jetpack
```

Comment:
With the future incorporation of the VideoPress package, seems more practical to install the dependencies from the root level.